--- ../src-base/minecraft/net/minecraft/server/network/NetHandlerHandshakeTCP.java
+++ ../src-work/minecraft/net/minecraft/server/network/NetHandlerHandshakeTCP.java
@@ -41,13 +41,13 @@
 
                 if (p_147383_1_.func_149595_d() > 5)
                 {
-                    chatcomponenttext = new ChatComponentText("Outdated server! I\'m still on 1.7.10");
+                    chatcomponenttext = new ChatComponentText("Outdated server! I\'m still on " + System.getProperty("nebula.version", "1.7.10"));
                     this.field_147386_b.func_150725_a(new S00PacketDisconnect(chatcomponenttext), new GenericFutureListener[0]);
                     this.field_147386_b.func_150718_a(chatcomponenttext);
                 }
                 else if (p_147383_1_.func_149595_d() < 5)
                 {
-                    chatcomponenttext = new ChatComponentText("Outdated client! Please use 1.7.10");
+                    chatcomponenttext = new ChatComponentText("Outdated client! Please use " + System.getProperty("nebula.version", "1.7.10"));
                     this.field_147386_b.func_150725_a(new S00PacketDisconnect(chatcomponenttext), new GenericFutureListener[0]);
                     this.field_147386_b.func_150718_a(chatcomponenttext);
                 }
